changelog/unreleased/1343

```diff
@@ -1,18 +0,0 @@
-Enhancement: Improved sidebar with account-wide search
```

Comment:
what happend to this change log entry

Reply:
it was a wrong entry because it was already released in version 12.2